src/BloomBrowserUI/bookEdit/js/synphonyApi.js

```diff
@@ -44,78 +38,64 @@ var SynphonyApi = (function () {
             }
         }
     };
-
     SynphonyApi.prototype.loadFromLangData = function (langData) {
         if (!this.source)
             this.source = new ReaderSettings();
-
         if (this.source.letters === '') {
             var sorted = langData.LanguageSortOrder.join(' ').toLowerCase().split(' ');
             sorted = _.uniq(sorted);
-
             this.source.letters = sorted.join(' ');
         }
     };
-
     SynphonyApi.fireCSharpEvent = function (eventName, eventData) {
-        //noinspection TaskProblemsInspection
-        var event = new MessageEvent(eventName, { 'view': window, 'bubbles': true, 'cancelable': true, 'data': eventData });
-        document.dispatchEvent(event);
+        fireCSharpAccordionEvent(eventName, eventData);
```

Comment:
The MessageEvent ctor works fine in javascript, but it's actually not declared anywhere in TypeScript-land, so it generates an error when called from TypeScript. Phillip and I both couldn't figure out how to tell TypeScript it was okay. Then I found that accordion.js (not generated from TS) had defined an identical function and was loaded and available. Calling that js function here avoids the TS issue and makes us only have the function in one place.